src/main/java/com/e451/model/ProductRecord.java

```diff
@@ -0,0 +1,87 @@
+package com.e451.model;
+
+import javax.persistence.*;
+
+@Entity()
+@Table(name = "product")
+public class ProductRecord {
+
+    @Id
+    @GeneratedValue(strategy=GenerationType.AUTO)
+    private Long productId;
+    private String commodityDesc;
+    private String subCommodityDesc;
+    private String manufacturer;
+    private String department;
+    private String brand;
+    @Column(name="curr_size_of_product")
+    private String currentSizeOfProduct;
+
+    public ProductRecord() {}
+
+    public ProductRecord(Long productId, String commodityDesc, String subCommodityDesc, String manufacturer, String department, String brand, String currentSizeOfProduct) {
```

Comment:
This is a similarly long line of code to the one in src/main/java/com/e451/domain/Product.java. Maybe split it across a few lines.